docs/apidiffs/current_vs_latest/opentelemetry-api.txt

```diff
@@ -1,2 +1,5 @@
 Comparing source compatibility of  against 
-No changes.
\ No newline at end of file
+===  UNCHANGED INTERFACE: PUBLIC ABSTRACT io.opentelemetry.api.common.Attributes  (not serializable)
+	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
+	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.Object get(io.opentelemetry.api.common.AttributeKey)
+		+++  NEW ANNOTATION: javax.annotation.Nullable
```

Comment:
This is fantastic that we can see this now! Also, I think this is fine, this was already documented as nullable, and documenting it officially via annotation is a good change.